### PR TITLE
I am a naughty commit!

### PR DESCRIPTION
**Pull request by hedss**
I have no change type!

### CODE REVIEW

**Comment by hedss** [comment]
I'm a strange comment.

**Review by hedss — COMMENTED** [review]
This is a comment on something.

**Comment by hedss** [comment]
Final comment.

**Review by ghost — COMMENTED** [review]
I am a different reviewer.

**Comment by ghost** [comment]
And I am a different comment.

File: dummy.js [hunk 1]
```diff
@@ -1 +1,2 @@
 console.log('Does nothing');
```
Comment (ghost) [inline dummy.js:1]: A different type of review.

**Review by ghost — CHANGES_REQUESTED** [review]

File: dummy.js [hunk 2]
```diff
@@ -1 +1,2 @@
...
+console.log('Now with added bad commit message');
```
Comment (ghost) [inline dummy.js]: Oh dear, it seems only certain review types are shown!

**Review by ghost — CHANGES_REQUESTED** [review]

File: dummy.js (hunk 1 above)
Comment (ghost) [inline dummy.js:1]: General review comment.

**Review by ghost — COMMENTED** [review]
General

**Review by ghost — COMMENTED** [review]
A generic comment not referencing the code.

**Review by ghost — CHANGES_REQUESTED** [review]
Some changes, please.

File: dummy.js (hunk 2 above)
Comment (ghost) [inline dummy.js]: Some other changes, please.

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**